tools/submission/submission-checker.py

```diff
@@ -558,9 +558,9 @@
 
 RESULT_FIELD_NEW = {
```

Comment:
Where is `RESULT_FIELD_NEW` used? I think this will break the backward compatibility when parsing v1.1 logs?

Reply:
Yes, it would break the backward compatibility. I will make this variable dependent on the config version